modules/classmethods.js

```diff
@@ -0,0 +1,25 @@
+import * as functions from './functions.js';
+
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+let myLibrary = [];
+// const storage = [];
```

Comment:
- [ ] OPTIONAL
There is a commented code block found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.